rem/DB.swift

```diff
@@ -147,17 +173,29 @@ class DatabaseManager {
                 return lastFrame[id]
             }
         } catch {
-            print("Error fetching last chunk ID: \(error)")
+            print("Error fetching last frame ID: \(error)")
+        }
+        return 0
+    }
+    
+    private func getLastChunksFramesIndex() -> Int64 {
+        do {
+            if let lastFrame = try db.pluck(chunksFramesView.order(chunksFramesIndex.desc)) {
+                return lastFrame[chunksFramesIndex]
+            }
+        } catch {
+            print("Error fetching last chunks frames Index: \(error)")
         }
         return 0
     }
     
     // Insert a new video chunk and return its ID
     func startNewVideoChunk(filePath: String) -> Int64 {
-        let insert = videoChunks.insert(self.filePath <- filePath)
+        let insert = videoChunks.insert(self.id <- currentChunkId, self.filePath <- filePath)
```

Comment:
Could remove auto increment but I don't see the harm in keeping the requirements of increasing and unique